templates/en/codelist.rst

```diff
@@ -9,10 +9,10 @@
 External URL: {{codelist_json.metadata.url}}
 {% endif %}
 
-{% if embedded %}
```

Comment:
if `embedded` property isn't used elsewhere, it can be removed from `gen.py`

Reply:
actually, it only needs to be removed from the 'context' - the os.path is used to determine the git url, which we still need: https://github.com/IATI/IATI-Standard-SSOT/blob/ac0a800f9070c67eac13770bcae5521934ea2419/gen.py#L552